packages/itwinui-css/backstop/tests/dialog.html

```diff
@@ -275,13 +331,21 @@ <h1>Dialog</h1>
       </div>
     </div>
 
-
     <script>
       function toggleDialog(dialogId) {
         let element = document.getElementById(dialogId);
         element.classList.toggle('iui-dialog-visible');
       }
 
+      const dialog = document.querySelector('#dialog-click');
+      // Clicking on menu-item should toggle selection
+      document.querySelectorAll('input').forEach((item) => {
+        item.addEventListener('click', () => {
+          // dialog.classList.value = `iui-toast-wrapper iui-placement-${item.value}`;
```

Comment:
You can remove this commented out line of code.

Reply:
fixed